packages/kbn-ftr-common-functional-services/services/saml_auth/saml_auth_provider.ts

```diff
@@ -110,7 +110,14 @@ export function SamlAuthProvider({ getService }: FtrProviderContext) {
           metadata: {},
           role_descriptors: roleDescriptors,
         });
-      expect(status).to.be(200);
+
+      if (status !== 200) {
+        throw new Error(
+          `Failed to create API key for '${role}' role: status code '${status}' and response: ${JSON.stringify(
```

Comment:
The `body` is never circular right? If so, it could fail.

Reply:
Good point, I change it in favour `response.text` property which seems to be always defined and of `string` type 8669309a373d09ce3aa7340e423843b0dc5846c1

```
   │1)    Alerting APIs
   │       Alerting rules
   │         "before all" hook for "should schedule task, run rule and schedule actions when appropriate":
   │
   │      Error: Failed to create API key for 'admin' role with response text: {"statusCode":400,"error":"Bad Request","message":"undefined is not iterable (cannot read property Symbol(Symbol.iterator))"}
   │       at Object.createM2mApiKeyWithRoleScope (saml_auth_provider.ts:115:15)
   │       at processTicksAndRejections (node:internal/process/task_queues:95:5)
   │       at Context.<anonymous> (rules.ts:34:19)
   │       at Object.apply (wrap_function.js:74:16)
```